services/worker-manager/src/provisioner.js

```diff
@@ -143,12 +143,10 @@ class Provisioner {
       };
 
       // Check the state of workers (state is updated by worker-scanner)
-      const fetch = async (size, offset) => await this.db.fns.get_workers(null, null, null, null, size, offset);
+      const fetch = async (size, offset) => await this.db.fns.get_non_stopped_workers(null, null, null, size, offset);
```

Comment:
It seems this got missed in #3509 / 7b7713c6e01833a76a4747c87c2588075bfcca1c?

Reply:
That PR was trying to tackle https://github.com/taskcluster/taskcluster/issues/3036 which only targeted the call in worker-scanner.